packages/ec-linear/examples/count_ones/main.rs

```diff
@@ -1,5 +1,14 @@
-#![allow(clippy::use_debug)]
-#![allow(clippy::arithmetic_side_effects)]
+// TODO: since inner attributes are unstable, we can't use rustversion here.
+// Once we revert this commit, this is proper again.
+#![allow(clippy::allow_attributes_without_reason)]
+#![allow(clippy::use_debug
+//    , reason = "Debug printing is useful for examples."
+)]
```

Comment:
no - if you take a look at the first commit, that is how it actually should be. we can't use custom interior attributes yet ( again, nightly only ) and as such we need to switch to allow temporarily here. This, again, will be un-commented once we revert the second commit in 2 months or so

Reply:
clarification: `expect` isn't custom so that works in the first commit ( since it comes from rust directly ) but `rustversion::attr` is and as such conditional compilation using that won't work, so as a "quick fix" ( till 1.81 is stable ) I just commented out the reason and switched it back to allow, and silenced the corresponding lint

Reply:
Thanks – that makes sense.